src/tools/Reporting/Reporting/Reporting.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

is this change necessary? 
```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```